mongodump/mongodump.go

```diff
@@ -38,22 +38,24 @@ type MongoDump struct {
 	InputOptions  *InputOptions
 	OutputOptions *OutputOptions
 
+	SkipUsersAndRoles bool
```

Comment:
interesting... Odd that they do that. I can see this getting very confusing for users. Not sure I have a better suggestion. Tools guys will obviously need to approve this one.